packages/angular-output-target/angular-component-lib/utils.ts

```diff
@@ -36,6 +36,28 @@ export const defineCustomElement = (tagName: string, customElement: any) => {
   }
 };
 
+/**
+ * The Angular property name that contains the object of metadata properties
+ * for the component added by the Angular compiler.
+ */
+const NG_COMP_DEF = 'ɵcmp';
+
+export const clearAngularOutputBindings = (cls: any) => {
```

Comment:
Can we add types here?

Reply:
The only type addition that I am familiar with that I could see applying here is something like:

```ts
type Clazz = new (...args: any[]) => unknown;

export const clearAngularOutputBindings = (cls: Clazz) => {
  ..
}
```

but this would only type `cls.prototype`. After that the inferred types are `any`, so it doesn't provide type safety where necessary. 

Also the calling implementation uses `any`: https://github.com/ionic-team/stencil-ds-output-targets/blob/34f652daf77703519d0036b4da7028756edab7b8/packages/angular-output-target/angular-component-lib/utils.ts#L63-L70 which doesn't really benefit this change.